console/console-init/ui/src/modules/iot-device/utils/TableFormatter.tsx

```diff
@@ -31,15 +39,22 @@ export const getTableCells = (row: IDevice) => {
         header: "id",
         title: (
           <span>
-            <Link to={`devices/${row.deviceId}/device-info`}>
-              {row.deviceId}
-            </Link>
+            <Link to={`devices/${deviceId}/device-info`}>{deviceId}</Link>
           </span>
         )
       },
       {
         header: "type",
-        title: <span>{deviceType}</span>
+        title:
+          deviceType === "N/A" ? (
```

Comment:
It would be better to wrap this code in function like renderDeviceType.

Reply:
done